components/prism-diff.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.diff = {
+	'coord': [
+		// Match all kinds of coord lines (prefixed by "+++", "---" or "***").
+		/^(\*{3}|-{3}|\+{3}).*$/m,
+		// Match "@@ ... @@" coord lines in unified diff.
+		/^@@.*@@$/m,
+		// Match coord lines in normal diff (starts with a number).
+		/^\d+.*$/m,
+	],
+
+	// Match inserted and deleted lines. Support both +/- and >/< styles.
+	'deleted': /^[-\<](?![-\<]).+$/m,
+	'inserted': /^[\+\>](?![\+\>]).+$/m,
```

Comment:
Why escape the angle brackets? Also `+` doesn’t need escaping inside a character class. 
Also, what’s the negative lookahead trying to prevent? What happens if the line starts with e.g. ++ or >>?